crates/hc_sandbox/src/ports.rs

```diff
@@ -46,19 +44,29 @@ pub async fn get_admin_ports(paths: Vec<PathBuf>) -> anyhow::Result<Vec<u16>> {
     Ok(ports)
 }
 
-pub(crate) async fn get_admin_api(port: u16) -> WebsocketResult<WebsocketSender> {
+pub(crate) async fn get_admin_api(
```

Comment:
In this PR I'll probably make some comment suggestions even if there wasn't a comment originally, not only because comments are good, but it's a test to see if I understand what I read.

```suggestion
/// Creates a [`WebsocketSender`] along with a task which simply consumes and discards
/// all messages on the receiving side
pub(crate) async fn get_admin_api(
```